cpp/src/parquet/metadata.cc

```diff
@@ -19,10 +19,13 @@
 
 #include <algorithm>
 #include <cinttypes>
+#include <csignal>
+#include <iostream>
 #include <ostream>
 #include <string>
 #include <string_view>
 #include <utility>
+#include <variant>
```

Comment:
Why are these new inclusions suddenly required?

Reply:
Done. I removed the code that used this include but forgot removing the include.